_posts/en/newsletters/2020-11-11-newsletter.md

```diff
@@ -0,0 +1,139 @@
+  a1link="https://bitcoincore.reviews/19055-2#l-131"
+
+  q2="What does the `#ifdef HAVE___INT128` code in the MuHash implementation do?"
+  a2="`#ifdef HAVE___INT128` is a C++ preprocessor directive. It ensures that code
+      contained within the `#ifdef` and `#endif` directives is only compiled on
+      platforms that support 128 bit integers. Since the MuHash code is
+      operating on numbers that are many times larger than built-in integer types
+      can handle, those numbers are broken down into multiple 'limbs', which are
+      combined to make the full number. Larger limbs means fewer operations and
+      therefore [better performance][muhash performance]. Support for 128 bit
+      integers allows us to use 64 bit limbs and safely multiply the numbers in
+      those limbs together (the product of two 64 bit numbers is a 128 bit number).
+      If we can't use 64 bit limbs, then 32 bit limbs are used instead."
+  a2link="https://bitcoincore.reviews/19055-2#l-190"
+
+  q3="What is 1437 mod 99?"
+  a3="1437 mod 99 = 51. This question was given as an example of an efficient way to
+      calculate the remainder of a number when divided by a modulus that is slightly
+      smaller than a power of the number base. In this example, 99 is slightly
+      smaller than 10^2, so to calculate the remainder we can add the bottom half
+      (37 in this case) to the top half times the difference of the modulus from the
+      base number power (14 * (10^2 - 99) in this case). The same technique is used
+      in the MuHash implementation and explains why the modulus was chosen as a
+      number slightly smaller than 2^3072."
+  a3link="https://bitcoincore.reviews/19055-2#l-246"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:none as of Sunday; harding to update or delete this section Tuesday
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [BOLTs #807][] Fail channel in case of high-S remote signature reception FIXME:dongcarl
+
+- [Eclair #1545][] adds a blockchain watchdog to detect eclipse attacks. Eclair
+  fetches block headers from multiple third-party sources to countercheck the
+  best chain provided by its peers. As described in [Newsletter #101][news101 time-dilation],
+  an attacker could use an eclipse attack to hide a malicious channel closing
+  attempt until the dispute period expires.
+
+- [LND #4735][] adds as `max_local_csv` configuration parameter that
```

Comment:
s/as/a/

maybe s/parameter/option/ like the writeup on LND #4701 in the next paragraph (and Bitcoin Core)